cflat.md

```diff
@@ -0,0 +1,100 @@
+> [name=Arbie, Yin, Guo, Ashima]
+
+## Information about the project
+
+### What is compiler explorer?
+
+Compiler Explorer is an interactive compiler exploration website. Edit code in C, C++, C#, F#, Rust, Go, D, Haskell,
+Swift, Pascal, ispc, Python, Java, or any of the other 30+ supported languages, and see how that code looks after being
+compiled in real time. Multiple compilers are supported for each language, many different tools and visualizations are
+available, and the UI layout is configurable (thanks to GoldenLayout).
+
+### What is our project is about?
+
+This project focuses on enhancing Compiler Explorer (CE) by adapting it to support in-browser compilers. The primary
```

Comment:
"This project" - do you mean Compiler Explorer?

It sounds like this should be a separate fork of Compiler Explorer, rather than something we maintain in the main CE repository.